includes/scripts.php

```diff
@@ -171,7 +171,7 @@ function edd_load_admin_scripts( $hook ) {
 	$css_dir = EDD_PLUGIN_URL . 'assets/css/';
 
 	// Use minified libraries if SCRIPT_DEBUG is turned off
-	$suffix  = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
+	$suffix  = ''; //( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
```

Comment:
This needs to be reverted. It will currently make it so no minified scripts load.

Reply:
I could have sworn I changed this back, sorry.